.github/workflows/check_on_push.yaml

```diff
@@ -0,0 +1,24 @@
+name: Run static analysis
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-static-analysis:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup luacheck
+        run: |
+          curl -L https://tarantool.io/installer.sh | sudo bash
+          sudo apt install -y tarantool-dev
```

Comment:
Just tarantool (without headers) should be okay. However I suggest to use [setup-tarantool](https://github.com/tarantool/setup-tarantool) for such simple workflows. I employs github's caching infrastructure to make pipelines incredibly fast.

Reply:
Yep, totally forget about this action. Now luacheck pipeline uses it.

Reply:
Nice!